readme.md

```diff
@@ -17,5 +17,5 @@ You can find some sample screenshots in the [backstop_data/bitmaps_reference](ba
 
 **Result**
 
-[UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/) <br>
-[UPDATE TEST REPORT LINK](https://<your_account>.github.io/<repo_name>/report/html_report/)
+[UPDATE DEMO LINK](https://mate-academy.github.io/layout_solutions/colored-blocks/) <br>
+[UPDATE TEST REPORT LINK](https://mate-academy.github.io/layout_solutions/colored-blocks/)
```

Comment:
please add here link your repo. Not the mate academy

Reply:
please change only <your_account> and <repo_name>. Keep `/report/html_report/` in the url